packages/core/src/components/Menu/MenuItem/components/MenuItemSubMenuIcon/MenuItemSubMenuIcon.module.scss

```diff
@@ -0,0 +1,32 @@
+.subMenuIconWrapper {
+  margin-left: var(--spacing-xs);
+
+  .divider {
+    height: 19px;
+  }
+
+  .splitSubMenuIcon {
+    width: 18px;
+    height: 18px;
+
+    &.disabled {
+      cursor: not-allowed;
+      color: var(--disabled-text-color);
+    }
+  }
+
+  .subMenuIcon,
+  .splitSubMenuIcon {
+    color: var(--icon-color);
+  }
+
+  .splitMenuItemIconButton {
+    width: 34px;
+    height: 30px;
+
+    // TODO Temp override for Tal changes - remove once token is changed
+    --primary-background-hover-color: rgba(103, 104, 121, 0.1);
```

Comment:
The token has changed, do we want to update it with the variable?

Reply:
I'm not sure
maybe as a first step let's just leave it as a refactor?
mind creating a task and assign to me? I'm not sure I'm familiar with the new variable

Reply:
for future reference:
https://monday.monday.com/boards/3532714909/pulses/6580960868

Thanks @talkor